.cirrus.yml

```diff
@@ -43,7 +43,7 @@ env:
     # Git commits to use while building dependencies into cache-images
     FEDORA_CNI_COMMIT: "412b6d31280682bb4fab4446f113c22ff1886554"
     CNI_COMMIT: "7480240de9749f9a0a5c8614b17f1f03e0c06ab9"
-    CONMON_COMMIT: "3e47d8dd45cdd973dbe256292d5e9c0bff195e56"
+    CONMON_COMMIT: "6f3572558b97bc60dd8f8c7f0807748e6ce2c440"
     CRIU_COMMIT: "c74b83cd49c00589c0c0468ba5fe685b67fdbd0a"
```

Comment:
FYI- this is my bad...these aren't used anywhere.  I've got a commit floating on a PR somewhere that just clobbers them all.